package/CHANGELOG

```diff
@@ -17,11 +17,14 @@ The rules for this file:
          lilyminium, daveminh, jbarnoud, yuxuanzhuang, VOD555, ianmkenney,
          calcraven，xiki-tempula, mieczyslaw, manuel.nuno.melo, PicoCentauri,
          hanatok, rmeli, aditya-kamath, tirkarthi, LeonardoBarneschi, hejamu,
-         biogen98, orioncohen
+         biogen98, orioncohen, thadanipaarth
 
   * 2.0.0
 
 Fixes
+  * Removed deprecated parameters `n_jobs` and `precompute_distances` of
+    sklearn.cluster.KMeans and Added new parameter `random_state` of 
```

Comment:
Could you please now remove this from the CHANGELOG?

Reply:
Done!